spec/gameSpec.js

```diff
@@ -0,0 +1,30 @@
+describe("Game", function () {
+    var game;
+    beforeEach(function () {
+        game = new Game();
+    });
+
+    it('Gutter Game', function () {
+        for (var i = 0; i < 20; i++) {
+            game.roll(0);
+        }
+        expect(game.score()).toEqual(0);
+    });
+
+    it('Score all ones', function () {
+        for (var i = 0; i < 20; i++) {
```

Comment:
Could you make a function that does this for loop for you instead of having to write it out each time?